Cargo.toml

```diff
@@ -372,13 +372,14 @@ libc = "0.2"
 pretty_assertions = "1"
 rand = "0.8"
 regex = "1.0"
-sha1 = { version="0.6", features=["std"] }
+sha1 = { version="0.10", features=["std"] }
 tempfile = "3.2.0"
 time = "0.1"
 unindent = "0.1"
 uucore = { version=">=0.0.11", package="uucore", path="src/uucore", features=["entries", "process"] }
 walkdir = "2.2"
 atty = "0.2"
+hex-literal = "0.3.1"
```

Comment:
I think it is a dev dependency, not a dependency which will ship in the product

Reply:
so, please move it in the correct section

Reply:
Thanks for the review. I don't think I understand what to do here. I had placed hex-literal in the [dev-dependencies] section of top-level Cargo.toml, which I *think* matches your requirement. 
(The dependency is only there to smooth conversion in the assertion in factor tests which directly call SHA1 themselves so only needed at build/CI time. The new SHA1 API doesn't offer the same convenience conversions as the old. There isn't a Cargo.toml deeper in the tests/ tree.)
Am I missing something else?

Reply:
my bad, sorry, i thought it was part of the regular dep. sorry